docs/docker.md

```diff
@@ -2,55 +2,103 @@
 
 ```bash
-# From your cozy-stack developement folder
-docker run -it --rm --name cozy-stack \
-    -v $(pwd):/go/src/github.com/cozy/cozy-stack \
-    -v $(pwd):/go/bin \
-    golang:1.12 \
-    go get -v github.com/cozy/cozy-stack
+# Get the files
+git clone https://github.com/cozy/cozy-stack
+cd cozy-stack/scripts
+
+cp env.sample .env
+# Edit the .env file to specify your own values for passwords, mail settings, ...
+# Avoid special characters in the passwords, it break the scripts. But make it long, something like the output of "uuidgen" will do.
+
+# Start the environment
+docker-compose up -d
+
+# Create a first instance
+source .env
+docker-compose exec -T stack bash -c "yes $COZY_ADMIN_PASSPHRASE | cozy-stack instances add --email you@$DOMAIN --passphrase AVerySecretPasswordHere YourFirstInstanceName.$DOMAIN --apps home,drive,settings,store,photos"
 ```
 
-## Publishing a new cozy-app-dev image
+Then heads to https://YourFirstInstanceName.stack.your.domain.
 
-We publish the cozy-app-dev image when we release a new version of the stack.
-See `scripts/release.sh` for details.
+Whenever done, here's the cleanup
 
+```bash
+docker-compose down
+sudo rm -rf volumes/
+```
 
-## Docker run and url name
+## The secure do it yourself way
 
-A precision for the app name :
+Provided you already got a server running with Docker installed, a reverse proxy running, a domain set up, and a wildcard certificate for `*.cozy.your.domain`, and set up your reverse proxy to redirect all incoming `*.cozy.your.domain` traffic to `localhost:8080`, you have to
 
-docker run --rm -it -p 8080:8080 -v "$(pwd)/build":/data/cozy-app/***my-app*** cozy/cozy-app-dev
+    docker run -d -e COUCHDB_USER=cozy -e COUCHDB_PASSWORD=cozy --name couch --volume $(pwd)/volumes/couchdb:/opt/couchdb/data couchdb
+    docker run -d -p 127.0.0.1:8080:8080 -p 6060:6060 --link couch --name stack -e LOCAL_USER_ID=$(id -u) -e LOCAL_GROUP_ID=$(id -g) --volume $(pwd)/volumes/stack:/var/lib/cozy/data cozy/cozy-stack
+
+An alternative to an expensive wildcard certificate can be Caddy Server as a reverse proxy. It's able to generate on demand certificates via the ACME protocol and LetsEncrypt. Meaning the first time you open an url, something like https://your-drive.cozy.your.domain, there'll be a few second lags for Caddy to grab the certificate, and you'll be good to go.
+
+## The unsecure all-in-one usage for tests
+
+This image is intended for development & local tests. It's compiled in development, hence some security features are disabled. Here's an HOWTO get started
+
+```bash
+# Start a CouchDB
+docker run -d -e COUCHDB_USER=cozy -e COUCHDB_PASSWORD=cozy --name couchdb couchdb
+
+# Start your cozy stack
+docker run -d -p 80:8080 -p 6060:6060 --link couchdb --name stack cozy/cozy-stack:development
```

Comment:
For development & local tests, I'd prefer to make it run on port 8080

Reply:
I'll remove most of the development & tests in favor of a link docs/client-app-dev/#with-docker